### ISSUE TITLE
Ferdi is now listed as Apple Silicon

### ISSUE THREAD

**Issue by vinayh** [ThatGuySam/doesitarm#867]
**The official name of the app**
Ferdi

**Is there a supported version available on a stable release channel?**
Yes

**Proposed New App Status**
✅ Yes, Native Apple Silicon Support

**Related Issue Tracker Links or discussion**
https://github.com/getferdi/ferdi/pull/1277

**An Official App Download Page**
https://getferdi.com/download/

**Additional context**
App is listed on download page for ARM architectures, and the application is verified as "Apple Silicon" on my M1 machine.

**Full Screenshot with the App and Activity Monitor(if self reporting) or [Silicon Info](https://github.com/billycastelli/Silicon-Info)**
<img width="949" alt="Screen Shot 2022-01-20 at 19 06 37" src="https://user-images.githubusercontent.com/931593/150405471-9f47b199-11ee-4b16-b6aa-5c4a0f326d83.png">

**Comment by Madhawa97** [comment]
This is already listed on the site. isn't it? https://doesitarm.com/app/ferdi/

**Comment by vinayh** [comment]
Yes, it appears to have been added after this issue was opened, when @ThatGuySam added the label. I believe this issue can now be closed.

**Comment by Madhawa97** [comment]
You're right. But what's weird is it's not listed on the README.md file. @vinayh So I'm not sure.

**Comment by ThatGuySam** [comment]
@Madhawa97 There's an automated bot that will add Apps for me. 

The README doesn't actually include all the apps.

**Comment by Madhawa97** [comment]
Cool. Thanks for the info.
